templates/stripes/stripe-table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+$caption = !empty($caption["html"]) ? $this->inline($caption["html"]) : null;
+$headersInFirstRow = !empty($headersInFirstRow) ? $headersInFirstRow : false;
+$headersInFirstColumn = !empty($headersInFirstColumn) ? $headersInFirstColumn : false;
```

Comment:
What if you name these `$hasHeadersInFirstColumn` so it sounds like a boolean and not like its own array?????